secrets/helper/helper.go

```diff
@@ -69,7 +69,11 @@ func setupGCPSSM(
 
 // InitECDSAValidatorKey creates new ECDSA key and set as a validator key
 func InitECDSAValidatorKey(secretsManager secrets.SecretsManager) (types.Address, error) {
-	if secretsManager.HasSecret(secrets.ValidatorKey) {
+	hasSecret, err := secretsManager.HasSecret(secrets.ValidatorKey)
+	if err != nil {
+		return types.ZeroAddress, err
+	}
```

Comment:
When the key file doesn't exist, HasSecret throws error then `InitECDSAValidatorKey` just throws error as well.